pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
+const MinHostPort int = 0
```

Comment:
actually, the error message can stay the same - we just have to allow to to pass.  We want a default value to be allowed but we don't want to document 0 as a meaningful value.